text-editor/text-editor-002-en.md

```diff
@@ -423,33 +425,46 @@ The component's view basically consist into a list of buttons :
 ```
 *Example of the bold / unbold button*
 
-As for the `TextEditor` component, the `TextToolbox` component is connected to
-the redux store so we can dispatch actions and get state values. On the above
-example, we're using `isBold` and `isTitle` properties which are coming from
-the state.
+Like the `TextEditor` component, the `TextToolbox` is connected to the redux
+store. On the above example, we're using `isBold` and `isTitle` properties which
+are coming from the state :
+
+```js
+export const INSTANCE_INITIAL_STATE = {
+  visible: false,
+  top: 0,
+  isLinkCreatorOpened: false,
+  range: null,
+  isBold: false,
+  isItalic: false,
+  isUnderline: false,
+  isTitle: false,
+  isQuote: false,
+  isLink: false,
+}
+```
+
 The `isTitle` flag is used to disable the  bold / unbold button when the
-selected text is a title as we always want titles to be bold.
+selected text is a title (we always want titles to be bold).
 The `isBold` flag is used to whether highlight or not the bold / unbold button
 when the selected text is aleady bold or not (this button behavior is similar
 to the one you'll find on any wording processor).
-When clicked, the button dispatches a `MUTATE` action which would trigger the
-selected text mutation.
+When clicked, the button dispatches a `MUTATE` action. That action is observed
+by an epic which will actually perform the selected text mutation.
 
-**How the flags are determined ?**
-Such flags are dermined by an epic. The epic is executed when we open the
-text toolbox, and also when we apply a text mutation. We simply grab the
-status of the selected text using the
+**How these flags values are determined ?**
+A dedicated epic is executed both when we open the text toolbox, and also when
+applying a text mutation. Selected text's state is automagically guessed using
 [`document.queryCommandState`](https://developer.mozilla.org/en-US/docs/Web/API/Document/queryCommandState)
-function. This function returns a boolean indicating whether the given command
-has been executed on the selection range, in other world, it indicates if the
-selection is of the given type :
+function. This function relies on CSS style and return a boolean indicating if
```

Comment:
I'm not sure, because when you highlight text stylized in bold by CSS, the queryCommandState('bold') return true :thinking: I'll remove that part as we're not that sure of how it works.